### PR TITLE
fixes search bar problem

### LINKED ISSUE #12: Problem search should be case-insensitive
It is supposed to be working in case-insensitive.

|`Two Sum`|`Two sum`|
|---|---|
|Simulator Screen Shot - iPhone 11 - 2019-10-01 at 13 30 35|Simulator Screen Shot - iPhone 11 - 2019-10-01 at 13 30 40|

### PR DESCRIPTION
**Pull request by ryonryon**
Signed-off-by: ryotogashi <ryo.togashi.ca@gmail.com>

### CODE REVIEW

**Comment by ryonryon** [comment]
fixes #12

File: LeetCodeAssistant/ProblemListTableViewController.swift [hunk 1]
```diff
@@ -188,4 +188,7 @@ class ProblemListTableViewController: UITableViewController, UISearchBarDelegate
         filteredAttemptedProblems = attemptedProblems
         filteredUnsolvedProblems = unsolvedProblems
     }
+    fileprivate func removeBrankAndLowercase(_ str: String) -> String {
+        return str.lowercased().replacingOccurrences(of: " ", with: "")
+    }
```
Comment (axross) [inline LeetCodeAssistant/ProblemListTableViewController.swift]: Why do you remove whitespaces? IMO `Two Sum` shouldn't be `Twosum`

**Review by axross — COMMENTED** [review]
🤔

**Comment by ryonryon** [comment]
@axross thank you for your review. I changed that.

**Review by axross — APPROVED** [review]
Thanks!

**Pull request merged**